samples/led-blink-multiple/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Threading;
+
+int lightTime = 1000;
+int dimTime = 200;
+int[] pins = new int[] {18, 24, 25};
+
+using GpioController controller = new();
+
+// configure pins
+foreach (int pin in pins)
+{
+    controller.OpenPin(pin, PinMode.Output);
+    controller.Write(pin, 0);
+    Console.WriteLine($"GPIO pin enabled for use: {pin}");
+}
+
+// enable program to be safely terminated via CTRL-c 
+Console.CancelKeyPress += (s, e) =>
+{
+    controller.Dispose();

```

Comment:
I'd recommend forcing the loop to finish here. If you do this it will also finish but might show up errors